### PR TITLE
Newsletters: add 183 (2022-01-19)

### PR DESCRIPTION
**Pull request by harding**
- [x] Releases, RCs, and topic updates @harding 
- [x] Client and services section @bitschmidty 
- [x] LDK 1013 @dongcarl

### CODE REVIEW

**Review by adamjonas — COMMENTED** [review]
Lookin' good

File: _posts/en/newsletters/2022-01-19-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  New Lightning Network node software [LNP Node][lnp node github] opened its [first LN channel][lnp tweet].
+  LNP Node is written in rust and supports LN as well as extensions to LN
+  collectively named "Bifrost" that enable future LN upgrades and "layer 3"
+  protocols on top of LN.
+
+- **Samourai adds taproot support:**
+  Samourai [v0.99.98][samourai v0.99.98] and Samourai [Dojo v1.13.0][samourai dojo v1.13.0]
+  (via the [bitcoinjs-lib][bitcoinjs-lib github] library) support P2TR addresses.
+
+FIXME:harding releases/RCs if any
```
[the remark this answers is not in the document]
Comment (harding) [reply _posts/en/newsletters/2022-01-19-newsletter.md:77]: @bitschmidty might be interesting if you want to add it last minute.

**Comment by harding** [comment]
Make edits (thanks!) and pushed topic updates.  Reviewed contributed sections (more thanks!).  No releases/RCs this week.

File: _posts/en/newsletters/2022-01-19-newsletter.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bitschmidty) [reply _posts/en/newsletters/2022-01-19-newsletter.md]: Thanks guys, adding a quick summary now

**Pull request merged**